index.js

```diff
@@ -123,7 +128,16 @@ export default class WebTorrent extends EventEmitter {
 
       this.dht.once('listening', () => {
         const address = this.dht.address()
-        if (address) this.dhtPort = address.port
+        if (address) {
+          this.dhtPort = address.port
+          if (this.natTraversal != null) {
+            try {
+              this.natTraversal.map({ publicPort: this.dhtPort, privatePort: this.dhtPort, protocol: 'udp', description: 'WebTorrent DHT' })
```

Comment:
this is async, so yo need a .catch() not a try-catch unless you make this function async and use await

Reply:
This has been amended to .catch the error as per the above comment.